test/unit/Optimize.ml

```diff
@@ -2190,6 +2190,11 @@ model {
   Fmt.strf "@[<v>%a@]" Program.Typed.pp mir |> print_endline ;
   [%expect
     {|
+      Info: integer division implicitly rounds to integer. Found int division: 1 / 2.
```

Comment:
This is fine for now; an artifact of the fact that we're using the full parser in our unit tests here. This will go away when we switch to #357